### DISCUSSION TITLE
Disable categories of rules

### DISCUSSION THREAD

**Discussion by jdecker79** [Security-Onion-Solutions/securityonion#3541] [Unsupported Versions]
In the documentation under Identifying rule categories.  It says "You should only run the rules necessary for your environment. So you may want to disable entire categories of rules that don’t apply to you." 

How do I disable an entire category of rules?  There is no mention in the documentation that I could find that tells how to do this.   For example we do not use VMware so I see no need to run rules in 0235-vmware_rules.xml.  My assumption is if I just delete the xml file it will reappear when the rules are update.

**Comment by presianbg** [comment]
Hi,

You may use `re` match like that:

```
  sids:
    disabled:
    - 're:ET POLICY'
    - 're:ET INFO'
```
under `idstools` section in your `manager.sls` pillar.

Hope this is what you are looking for.
If you still having alerts for the disabled categories, then you have to check the flowbit deps.
